R/ioslides_presentation.R

```diff
@@ -18,6 +18,7 @@ ioslides_presentation <- function(logo = NULL,
                                   keep_md = FALSE,
                                   lib_dir = NULL,
                                   md_extensions = NULL,
+                                  template = NULL,
```

Comment:
In the `html_document` format the `template` argument goes right after `mathjax`, could we do the same here?